tests/pipelines/ddpm/test_ddpm.py

```diff
@@ -68,7 +68,7 @@ def test_inference(self):
         assert np.abs(image_slice.flatten() - expected_slice).max() < 1e-2
         assert np.abs(image_from_tuple_slice.flatten() - expected_slice).max() < 1e-2
 
-    def test_inference_predict_epsilon(self):
+    def test_inference_deprecated_predict_epsilon(self):
         deprecate("remove this test", "0.10.0", "remove")
         unet = self.dummy_uncond_unet
         scheduler = DDPMScheduler(predict_epsilon=False)
```

Comment:
This test is now failing because `DDPMScheduler` is initialized with the deprecated `kwarg` and then it's replaced with the default `prediction_type` in `register_to_config`. @patrickvonplaten should we address this in a followup PR?